src/rmkit/ui/button.cpy

```diff
@@ -38,12 +38,16 @@ namespace ui:
       Button::key_ctr++
       self.text = t
       self.textWidget = make_shared<Text>(x, y, w, h, t)
-      self.set_justification(ui::Text::JUSTIFY::CENTER)
       #ifdef DEV
       debug self.text, "=", input::get_key_str(self.key)
       #endif
+      self.set_style(DEFAULT_STYLE)
       return
 
+    void set_style(const Stylesheet & style):
+      Widget::set_style(style)
+      self.textWidget->set_style(Stylesheet::Inherited(self.style).text_style())
```

Comment:
I ended up going with a `Widget::set_style()` function instead of editing styles directly, but also happy to stick with directly updating styles (and copying to children in before_render) if you'd prefer.

Since applying a stylesheet is still a function call (`Stylesheet::apply()`), the syntax isn't all that different, but `set_style` feels like it keeps things a little more encapsulated to me:
```cpp
// with set_style
myButton.set_style(myStylesheet);
// directly editing the style member
myStylesheet.apply(myButton.style);
```

Reply:
set_style is fine, maybe adding `operator +=(Stylesheet)` could be a nice shorthand, like: 

```
myButton += myStylesheet; // calls myButton.set_style(...)
```